packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -611,43 +617,30 @@ export function TransferPanelMain({
 
     function modifyOptions(selectedChainId: ChainId, direction: 'from' | 'to') {
       // Add L1 network to the list
-      return [l1.network, ...options]
-        .filter(option => {
-          // Remove the origin network from the destination list for contract wallets
-          // It's done so that the origin network is not changed
-          if (
-            isSmartContractWallet &&
-            direction === 'to' &&
-            option.id === from.id
-          ) {
-            return false
-          }
-          // Remove selected network from the list
-          return option.id !== selectedChainId
-        })
-        .map(option => {
```

Comment:
Removed `.map` because we don't need to disable ona and nova selections anymore.